src/frontend/components/Chat/SharedChatComponents/ChatMessageContent/index.tsx

```diff
@@ -6,22 +6,26 @@ import { theme } from 'utils/theme/theme';
 
 interface ChatMessageContentProps {
   messageContent: string;
+  standalone?: boolean;
```

Comment:
I don't find `standalone` is an appropriate naming (yet I have nothing better) but I understand the purpose of this props only because of the PR context
eventually add documentation ?

Reply:
I changed how it works. Now messages are groupes and displayed by a ChatGroupMessageItem component, which uses itself ChatMessage component to displays message text content.